src/puya/awst_build/main.py

```diff
@@ -54,6 +54,10 @@ def transform_ast(
                 module._fullname = embedded_src.puya_module_name  # noqa: SLF001
                 module_awst = ModuleASTConverter.convert(ctx, module)
                 result[module.name] = module_awst
+            elif module_name.startswith("puya_util_"):
```

Comment:
Is this still required? Embedded modules should have been caught in the previous elif block

Reply:
Nope don't think it is. I'll remove it